src/toggle-switch/toggle-switch.scss

```diff
@@ -16,35 +16,32 @@
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
-  > input {
+  > input[type='checkbox'] {
```

Comment:
Zoomed on 125% on chrome, this is what I get
![image](https://user-images.githubusercontent.com/81580355/151775923-882130ef-4798-44c9-8ea1-5e6268b3b064.png)

90% look like that
![image](https://user-images.githubusercontent.com/81580355/151776019-08e026e1-be16-41df-a522-df41121323f5.png)


But not sure anymore, if thats my eyes or its not aligned :D